Keas.Mvc/Controllers/HomeController.cs

```diff
@@ -6,19 +6,38 @@
 using Microsoft.AspNetCore.Mvc;
 using Keas.Mvc.Models;
 using Microsoft.AspNetCore.Authorization;
+using Keas.Core.Data;
+using Keas.Mvc.Services;
 
 namespace Keas.Mvc.Controllers
 {
     [Authorize]
     public class HomeController : SuperController
     {
+        private readonly ApplicationDbContext _context;

```

Comment:
@jcubbage the formatting here is kinda strange all over the file.  The private vars aren't aligned, the if statement has no parenthesis spacing and the content inside the if statement isn't aligned either.  If this is in VS code maybe make sure you have the C# code extension installed and run the formatter

Reply:
Better?